xml/docu_styleguide.webwriting.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec-webwriting">
+ <title>Writing for the Web</title>
+
+ <para>
+  Create engaging and informative content that helps your audience and is
+  optimized for the web.
+ </para>
+
+ <sect2 xml:id="sec-topical">
+  <title>Topical structure</title>
+  <para>
+   The most important thing to do with your web copy is to help users get
+   answers to their questions as soon as possible. To achieve this, we
+   recommend the modular approach of topic-based authoring where documentation
+   is created and maintained in chunks (topics).
```

Comment:
maybe say something like: "...is created and maintained in chunks, continued to be refereed to as topic."